EasyPost/Constants.cs

```diff
@@ -117,20 +118,22 @@ public static class ErrorMessages
         /// <summary>
         ///     Common carrier account type groups.
         /// </summary>
-        public static class CarrierAccountTypes
+        public static class CarrierAccounts
         {
-            public const string FedExAccount = "FedexAccount";
-
-            public const string UpsAccount = "UpsAccount";
```

Comment:
issue: you mention "no concern for breaking change issue"; however, this is already in the wild and we can't remove it. For example, if someone has already updated to the last version and starts using these or updates to that version on their way to this upcoming one, this is a breaking change to have them released.

Please instead leave these here with a note to deprecate them in the next version (a TODO for easy searching) and we can remove them whenever we have another major release. It won't hurt anything to retain them. That way it's backwards compatible while still adding your new enums.

Reply:
While I disagree, I will re-add them, marked as deprecated.

Reply:
Thank you, this would unfortunately break the semver contract without keeping them.